Maskbook/Scene/Wallet/WalletSend/SendTransactionViewController.swift

```diff
@@ -112,31 +123,43 @@ class SendTransactionViewController: BaseViewController {
             make.left.equalTo(22.5)
             make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(24)
         }
-        
-        view.addSubview(nextButton)
-        nextButton.snp.makeConstraints { make in
-            make.right.equalTo(-22.5)
-            make.height.equalTo(54)
-            make.width.equalTo(68)
-            make.top.equalTo(toAddressLabel.snp.bottom).offset(8)
-        }
-        
+            
         view.addSubview(enterAddressTextField)
         enterAddressTextField.snp.makeConstraints {make in
             make.left.equalTo(toAddressLabel)
-            make.top.bottom.equalTo(nextButton)
-            make.right.equalTo(nextButton.snp.left).offset(-12)
+            make.top.equalTo(toAddressLabel.snp.bottom).offset(8)
+            make.right.equalTo(-22.5)
+            make.height.equalTo(54)
         }
         
         view.addSubview(tableView)
-
         tableView.snp.makeConstraints { make in
-            make.top.equalTo(enterAddressTextField.snp.bottom).offset(16)
+            make.top.equalTo(enterAddressTextField.snp.bottom).offset(4)
             make.left.right.equalTo(view)
             make.bottom.equalTo(view.safeAreaLayoutGuide)
         }
-
         tableView.dataSource = dataSource
+        
+        view.addSubview(nextButton)
+        nextButton.snp.makeConstraints { make in
+            make.right.equalTo(-23)
+            make.left.equalTo(23)
+            make.height.equalTo(54)
+            make.bottom.equalTo(view.safeAreaLayoutGuide.snp.bottom).offset(-24)
+        }
+    }
+    
+    func setTableViewSource() {
+        var snapshot = NSDiffableDataSourceSnapshot<Section, Item>()
```

Comment:
Is it more appropriate to create an `NSDiffableDataSourceSnapshot` in the `ViewModel`?